File: python_semicolon/assignment/java/Myshop.java
import java.util.Scanner;
public class Myshop{

	public static double shoping(double amount, double payment){

		double change= 0;
		change = payment - amount;

		return change;

}




	public static void main(String...args){

	Scanner sc= new Scanner(System.in);
	System.out.println("Enter the cost amount of the goods the customer is buying");
	double amount = sc.nextDouble();
	System.out.println("Enter the amount the customer pay in");
	double pay = sc.nextDouble();
	double change = shoping(amount, pay);
	System.out.printf("your change is %.1f penisis", change);

	

}


}

File: python_semicolon/assignment/myshop.py
def shop(amount, pay):
	change = 0
	change = (amount - pay)
	changes = change * -1

	return changes



item = int(input("Enter the amount of the goods customer is buying \n"))
payment = int(input("Enter the amount the customer give to you \n"))
pay = shop(item, payment)
print(f" the customer change is {pay} pennies")
